test/e2e/flowaggregator_test.go

```diff
@@ -221,8 +220,8 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 
 	// Polling to make sure all the data records corresponding to the iperf flow
 	// are received.
-	err = wait.Poll(250*time.Millisecond, collectorCheckTimeout, func() (bool, error) {
-		rc, collectorOutput, _, err := provider.RunCommandOnNode(controlPlaneNodeName(), fmt.Sprintf("kubectl logs --since=%v ipfix-collector -n antrea-test", time.Since(timeStart).String()))
+	err = wait.PollImmediate(500*time.Millisecond, flowAggregatorExportInterval, func() (bool, error) {
+		rc, collectorOutput, _, err := provider.RunCommandOnNode(controlPlaneNodeName(), fmt.Sprintf("kubectl logs --since=%v --pod-running-timeout=%v ipfix-collector -n antrea-test", time.Since(timeStart).String(), flowAggregatorExportInterval.String()))
```

Comment:
trying to understand what `--pod-running-timeout` is for here. In `setupTestWithIPFIXCollector` we already wait for the collector Pod to be running.

Reply:
As we running many test cases, ipfix collector could have crashed. This was more of an extra gating to cut down the time in those failure scenarios.

Reply:
ok it would be good to add a comment to explain the choice of `--pod-running-timeout`